lua/roslyn/lsp.lua

```diff
@@ -82,6 +86,50 @@ function M.start(bufnr, root_dir, on_init)
         end
     end
 
+    config.on_attach = function(client, attach_bufnr)
+        local original_request = client.request
+        local last_diagnostic = nil
+        local default_handler = roslyn_config.config.handlers["textDocument/diagnostic"]
```

Comment:
`roslyn.config.handlers` might be nil if the user doesn't send in any `handlers` in the setup. We should just set `handlers` to an empty table in the default config in `config.lua`